tests/conftest.py

```diff
@@ -123,6 +123,27 @@ def adata_mini():
     return read_csv(f"{TEST_DATA_PATH}/dataset1.csv", columns_obs_only=["glucose", "weight", "disease", "station"])
 
 
+@pytest.fixture
+def diabetes_130_fairlearn_sample():
+    adata = ep.dt.diabetes_130_fairlearn(
+        columns_obs_only=[
+            "race",
+            "gender",
+            "age",
+            "readmitted",
+            "readmit_binary",
+            "discharge_disposition_id",
+        ]
+    )[:200]
+    return adata
+
+
+@pytest.fixture
+def mimic_2_sample():
```

Comment:
Maybe name that more clearly? Above you also have `dataset_sample` but you subsample down to 200 observations. Here, you pick a few columns. Maybe get the column names into the name of the fixture.